client/lib/directly/README.md

```diff
@@ -72,11 +72,8 @@ which set of configs will be used.
 on `initialize()`. This is built-in to the system and we don't have much direct control
 over it. There may be mitigation strategies if this becomes undesirable.  
 
-- Directly's code uses some poorly-supported JavaScript functions like String.startsWith(),
-but it offers a fallback if a browser doesn't support it. Unfortunately this usage triggers
-errors in `development` from [`lib/wrap-es6-functions`](../wrap-es6-functions). The errors
-are innocuous and do not prevent the widget from working. One proposed solution is to
-re-enable Babel's polyfill which would remove the need for the error warnings.
+- Directly's code uses some poorly-supported JavaScript functions like String.startsWith().
```

Comment:
This entire bullet item should be removed.

Directly's code is entirely third-party, so it's not being compiled by Calypso. So the Babel polyfill in Calypso actually makes no difference to the behavior of Directly.

The only reason this bullet was here was to note that the console errors `wrap-es6-functions` threw on Directly code were unavoidable. Since those errors have been removed, there's no need for this bullet any more.